Intro-to-Solidity.md

```diff
@@ -142,7 +142,7 @@ contract Conditions {
 
     /*
         Name of the function is foo.
-        It takes in  uint and returns an uint.
+        It takes in a uint and returns a uint.
```

Comment:
Hey @EvgeniiaVak,
If the word begins with a vowel, then we use the indefinite article `an`, else `a`. You can learn more about it [here](https://owl.purdue.edu/owl/general_writing/grammar/articles_a_versus_an.html).
Could you change these two to `an`? The rest of the edits are correct.

Reply:
Ok, I replaced in a couple of other places there was `a uint` also for consistency. I guess that depends on how one reads the `u`.